packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -336,6 +356,7 @@ export class Portfolio extends PortfolioBase {
     if (props.tagOptions !== undefined) {
       this.associateTagOptions(props.tagOptions);
     }
+    cdk.Aspects.of(this).add(new PortfolioAssetBucketSharer());
```

Comment:
Oh I see. I don't think you need this. You can do the same operation eagerly, as the construct is being mutated.